examples/js/api-lifecycle.test.js

```diff
@@ -0,0 +1,173 @@
+'use strict'
```

Comment:
The format of this file doesn't conform to the standard.js conventions. I recommend either running the `lint-fix` script in `package.json` or removing standard.js and reformatting _all_ the files in the repo according to some other standard.